### ISSUE TITLE
This is not supported yet, sorry :( We currently only support mini-batch training for adjacency matrix representation but not for incidence matrix representation (although it is possible that we add such feature in the future).

### ISSUE THREAD

**Issue by ShuaiWang97** [pyg-team/pytorch_geometric#7365]
This is not supported yet, sorry :( We currently only support mini-batch training for adjacency matrix representation but not for incidence matrix representation (although it is possible that we add such feature in the future).

_Originally posted by @rusty1s in https://github.com/pyg-team/pytorch_geometric/issues/2618#issuecomment-846538801_

**Comment by rusty1s** [comment]
Yes, this is exactly what you would need to do (re-implementing `Data.__inc__`. We have a longer-term roadmap planed for this (see #7312). Feel free to join the discussion there, we are always looking for contributors :)

**Comment by SauravMaheshkar** [comment]
@CodeTal Is it safe to assume that the current implementation supports sampling using the various implementations in `torch_geometric.sampler` ?
